chaossaltstack/machine/scripts/kill_processes.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [ -z "$process_name" ]; then
+    echo "Please provide process name."
+    exit 1
+fi
+
+experiment=$(killall -9 $process_name)
```

Comment:
would it make sense to support killing with a different flag? maybe send a SIGINT? could this be a parameter?

Reply:
ok, let me try and test one AWS as well